src/libcollections/btree/map.rs

```diff
@@ -349,7 +349,7 @@ pub struct OccupiedEntry<'a, K: 'a, V: 'a> {
 }
 
 // An iterator for merging two sorted sequences into one
-struct MergeIter<K, V, I: Iterator<Item=(K, V)>> {
+struct MergeIter<K, V, I: Iterator<Item = (K, V)>> {
```

Comment:
I don't mind this, but it surprises me - most of the code I've seen in the Rust codebase does not use spaces around `=` in bounds on associated types.